### DISCUSSION TITLE
Welcome to umbrellaplug.github.io Discussions!

### DISCUSSION THREAD

**Discussion by umbrellaplug** [umbrellaplug/umbrellaplug.github.io#51] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  Going to start trying to use this section more to better communicate about the project and take suggestions and requests.

**Comment by wwteele** [comment]
Explain Source and Filters Settings

**Reply by umbrellaplug** [reply]
Hey, sorry for the delay in response. I will add some explanations of the filters and some videos later today.

**Comment by wwteele** [comment]
Great, thank you so much.

**Reply by umbrellaplug** [reply]
> Great, thank you so much.

#83

**Reply by umbrellaplug** [reply]
Sorry, been really busy with a new feature that was requested and life in general. I went ahead and made a discussion to try to explain the new filtering and sorting settings. I will work on getting a video done once I catch up on some of my requests.

**Comment by wwteele** [comment]
No worries, thanks for the update!
